internal/npm_install/generate_build_file.js

```diff
@@ -425,40 +466,23 @@ function listFiles(rootDir, subDir = '') {
             return isDirectory ? files.concat(listFiles(rootDir, relPath)) : files.concat(relPath);
           },
           [])
+      // Files with spaces (\x20) or unicode characters (<\x20 && >\x7E) are not allowed in
+      // Bazel runfiles. See https://github.com/bazelbuild/bazel/issues/4327
+      .filter(f => !f.match(/[^\x21-\x7E]/))
```

Comment:
Using RegExp.prototype.test() here is more efficient, since match() returns an array whereas test() returns boolean value.

Reply:
👍

Reply:
Good catch thanks